src/static/skins/colibris/pad.js

```diff
@@ -1,4 +1,32 @@
 function customStart()
 {
   $('#pad_title').show();
+
+  var showMoreIconBtn = $('<span class="show-more-icon-btn"></span>')
+  showMoreIconBtn.click(function() {
+    $('.toolbar').toggleClass('full-icons');
+    $('#editorcontainer').css('top', $('.menu_left').height() + 1 + 'px');
+  })
+  $('.toolbar').append(showMoreIconBtn)
+
+  var timer;
+  // on resize end
+  window.onresize = function(){
+    clearTimeout(timer);
+    timer = setTimeout(checkAllIconAreDisplayedInToolbar, 100);
+  };
+  setTimeout(checkAllIconAreDisplayedInToolbar, 300);
+  setTimeout(checkAllIconAreDisplayedInToolbar, 600);
```

Comment:
Relying on timing for adjusting the layout is too risky. We are sure that there will be quirks with different device speed / browser combinations.